include/bitcoin/bitcoin/utility/png.hpp

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright (c) 2011-2015 libbitcoin developers (see AUTHORS)
+ *
+ * This file is part of libbitcoin.
+ *
+ * libbitcoin is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License with
+ * additional permissions to the one published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option)
+ * any later version. For more information see LICENSE.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+#ifndef LIBBITCOIN_PNG_HPP
+#define LIBBITCOIN_PNG_HPP
+
+#include <cstdint>
+#include <istream>
+#include <bitcoin/bitcoin/compat.hpp>
+#include <bitcoin/bitcoin/define.hpp>
+#include <bitcoin/bitcoin/utility/data.hpp>
+
+#ifdef WITH_PNG
+#include "png.h"
+#endif
+
+namespace libbitcoin {
+
+#ifdef WITH_PNG
+
+namespace png {
+
+typedef uint32_t color[4];
+
+BC_CONSTEXPR uint32_t dpi = 72;
```

Comment:
Especially given that `inches_per_meter` is used below, this should be `dots_per_inch`. We generally try to avoid even common acronyms as a matter of style unless there is a compelling reason to accept one. There doesn't seem to be a compelling reason to accept this one, although `png` and `qrcode` both seem appropriate.